ipywidgets/widgets/widget_templates.py

```diff
@@ -70,6 +78,11 @@ def __init__(self, **kwargs):
         super(LayoutTemplate, self).__init__(**kwargs)
         self._copy_layout_props()
 
+    @observe(All)
+    def _delegate_to_layout(self, change):
+        "delegate the trait types to their counterparts in self.layout"
+        setattr(self.layout, change['name'], change['new']) # pylint: disable=no-member
```

Comment:
This will also set Layout's `merge` property, which isn't a trait, but it passes silently, e.g.
```python
assert not hasattr(app.layout, 'merge')
app.merge = False
assert hasattr(app.layout, 'merge')
```

Reply:
good point. I fixed that.